js/src/Ice/IdentityToString.js

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+import { ToStringMode } from "./ToStringMode.js";
+import { StringUtil } from "./StringUtil.js";
+
+/**
+ * Converts an object identity to a string.
+ *
+ * @param ident The object identity to convert.
+ *
+ * @param toStringMode Specifies if and how non-printable ASCII characters are escaped in the result.
+ *
+ * @return The string representation of the object identity.
+ **/
+export function identityToString(ident, toStringMode = ToStringMode.Unicode) {
```

Comment:
Move to a separate file to avoid circular dependencies with `LocalExceptions.js`